app/layouts/default.njk

```diff
@@ -1,9 +1,18 @@
 <!DOCTYPE html>
 <html class="{% block pageClass %}{% endblock %}" lang="en">
   <head>
+    <meta property="og:image" content="https://d33wubrfki0l68.cloudfront.net/8afee13b9e497a643b836562965a1a49e580e099/4bab0/images/cover.png">
+    <meta property="og:image:type" content="image/png">
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://megananderson.design/"/>
+    <meta property="og:title" content="Megan Anderson" />
```

Comment:
Maybe, "Megan Anderson: Product Designer"

Reply:
I think since designer is in the description is doesn't need that

Reply:
That makes sense